### DISCUSSION TITLE
Basic question about VAFs calculation

### DISCUSSION THREAD

**Discussion by fraterenz** [PoisonAlien/maftools#641] [Q&A] [answered]
Hi all,

thank you very much for this super cool package! 

I'm pretty new to the field and I had a question about VAF. Can I compute the VAF by using the MAF columns `t_ref_count` and `t_alt_count`, where the 2 columns are defined as the [NCI GDC documentation page](https://docs.gdc.cancer.gov/Data/File_Formats/MAF_Format/#protected-maf-file-structure)? More precisely, can I say that `VAF = t_alt_count / t_ref_count * 100` ? 

thank you very much and my apologies for the basic question...

Francesco

**Comment by PoisonAlien** [comment, marked as answer]
Hello Francesco,

You could do that manually to the data slots of MAF object.

e.g;
```r
laml@data[, t_vaf := t_alt_count /( t_alt_count+t_ref_count) * 100] # main data slot with non-syn variants
laml@maf.silent[, t_vaf := t_alt_count / (t_alt_count+t_ref_count) * 100] # data slot with syn variants
``` 

But regardless some functions that use VAF information will calculate this automatically for you in the absence of VAF column.

**Reply by fraterenz** [reply]
great, thank you!
